src/client/entity-editor/common/entity.tsx

```diff
@@ -41,7 +42,8 @@ function Entity(
 			}
 			{nameComponent}
 			{
-				disambiguation &&
+				author ? <span className="disambig margin-left-0-3"><small>({author})</small></span> :
+					disambiguation &&
```

Comment:
It's not going to be ideal display-wise, but it would be best to show both the disambiguation followed by the author name.

Reply:
How about name -author (disambiguation)?